src/migrations/20201019192018-add-businessPhysicalAddress.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('businessPhysicalAddress', {
```

Comment:
Can the same address belong to multiple businesses? This relationship table should exist only for n:m relationships.